man.Rmd

```diff
@@ -92,9 +131,9 @@ Here's what the result looks like in RStudio:
 knitr::include_graphics("images/man-add.png", dpi = 220)
 ```
 
-The default help-seeking process looks inside **installed** packages, so to see the documentation during development, devtools overrides the usual help functions with modified versions that know to consult the **source** package.
-To activate these overrides, you need to run `devtools::load_all()` once.
-So if the development documentation doesn't appear, you may need to load your package first.
+The default help-seeking process looks inside **installed** packages, so to see your package's documentation during development, devtools overrides the usual help functions with modified versions that know to consult the **source** package.
+To activate these overrides, you'll need to run `devtools::load_all()` at least once.
+If it feels like your edits to the roxygen comments aren't having an effect, double check that you have actually regenerated the `.Rd` files with and that you've loaded your package.
```

Comment:
Mention that you should see "Rendering development documentation"?

Reply:
done